docs/source/tutorials.rst

```diff
@@ -1,82 +1,55 @@
 Tutorials
 =========
 
```

Comment:
We could add something like Prerequisite: installation, with a link. Not sure how this works in rst, though.

Reply:
I can reference to it, yes!